mapper/src/cell_attach.rs

```diff
@@ -0,0 +1,298 @@
+use super::gps::{altitude, hdop, latlon, speed, time};
+use super::*;
+
+#[derive(Serialize, Debug, Clone, PartialEq)]
+pub struct CellAttach {
+    // This allows us to detect censorship efforts. It can roll over.
```

Comment:
What exactly does roll over mean for attach_counter?

Reply:
Each attach attempt gets counted. We probably won't ever do more than 4.3M attaches, but you never know :shrug: